### PR TITLE
Remove menu in the deputy space and change deputy mail

### PR DESCRIPTION
**Pull request by illusionOfParadise**

### CODE REVIEW

File: templates/deputy/_layout.html.twig [hunk 1]
```diff
@@ -16,33 +16,6 @@
             </header>
 
             <div class="l__wrapper procuration-manager__content">
-                {% set pages = [
```
Comment (ottaviano) [inline templates/deputy/_layout.html.twig]: do we need to block the controller actions too?

Comment (illusionOfParadise) [reply templates/deputy/_layout.html.twig]: Currently in the Controller we use canary mode and these actions are blocked in the prod.
When we will open deputy space for prod we will keep the canary mode for these cached actions.

Comment (illusionOfParadise) [reply templates/deputy/_layout.html.twig]: It's possible that actions will be used lately.

Comment (ottaviano) [reply templates/deputy/_layout.html.twig]: 👌

**Pull request merged**